### PR TITLE
test: add e2e test for the key ceremony

### PR DESCRIPTION
**Pull request by beagleknight**
This adds a new E2E test for the key ceremony. I added some plugins for cypress:
- To easily upload fixture files, so
 I can upload the private keys for the trustees
- The `testing-library` plugin because it is awesome.

You can see the test in action below:
video

~⚠️ The test is a bit flaky at the moment and we need to find a workaround soon.~ **Update:** the test is no longer flaky!

### CODE REVIEW

**Comment by beagleknight** [comment]
Back to draft because the test is not as stable as I thought!

**Review by orlera — APPROVED** [review]
⌛

**Review by agustibr — APPROVED** [review]

File: decidim-bulletin_board-app/spec/cypress/integration/sandbox/key_ceremony.page.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  performKeyCeremony({ title }, trustees) {
+    cy.findByText("key_ceremony").should("be.visible");
+    cy.findByText("Perform key ceremony").click();
+    cy.findByText(`Key ceremony for ${title}`).should("be.visible");
+
+    trustees.forEach(({ unique_id, name }) => {
+      cy.findByText(name)
+        .parent("tr")
+        .within((trusteeRow) => {
+          // Ensure that the button is present before starting uploading the keys
```
Comment (slickepinne) [inline decidim-bulletin_board-app/spec/cypress/integration/sandbox/key_ceremony.page.js:30]:
```suggestion
          // Ensure that the button is present before starting to upload the keys
```

File: decidim-bulletin_board-app/spec/cypress/support/commands.js [hunk 2]
```diff
@@ -23,3 +23,5 @@
 //
 // -- This is will overwrite an existing command --
```
Comment (slickepinne) [inline decidim-bulletin_board-app/spec/cypress/support/commands.js:24]:
```suggestion
// -- This will overwrite an existing command --
```

**Review by slickepinne — APPROVED** [review]
Nice! 🎉

**Pull request merged**